### PR TITLE
Fix illegal syntax & illegal variable references

### PR DESCRIPTION
**Pull request by SwadicalRag**
it actually works (albeit after these changes) still btw if ur wondering

### CODE REVIEW

File: lua/websocket/server.lua [hunk 1]
```diff
@@ -201,7 +202,7 @@ function websocket.CreateServer(addr, port, queue) -- non-blocking and max queue
 		connections = {}
 	}, SERVER)
 	server.socket:settimeout(0)
-	server.socket:bind(addr, port)
+	print(server.socket:bind(addr, port))
```
Comment (danielga) [inline lua/websocket/server.lua]: Can you remove this print please?

File: lua/websocket/utilities.lua [hunk 2]
```diff
@@ -38,7 +38,7 @@ end
 
 function utilities.HTTPHeaders(request)
 	assert(type(request) == "table", "parameter #1 is not a table")
-	assert(request[1] and find(request[1], ".*HTTP/1%.1") ~=, "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
+	assert(request[1] and find(request[1], ".*HTTP/1%.1"), "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
```
Comment (danielga) [inline lua/websocket/utilities.lua]: Can you change this back to `~= nil` to keep the style consistent?

**Review by danielga — CHANGES_REQUESTED** [review]

File: lua/websocket/server.lua (hunk 1 above)
Comment (SwadicalRag) [reply lua/websocket/server.lua]: can't believe I left this in, sorry!

File: lua/websocket/utilities.lua (hunk 2 above)
Comment (SwadicalRag) [reply lua/websocket/utilities.lua]: done!

File: lua/websocket/utilities.lua [hunk 3]
```diff
@@ -38,7 +38,7 @@ end
 
 function utilities.HTTPHeaders(request)
 	assert(type(request) == "table", "parameter #1 is not a table")
-	assert(request[1] and find(request[1], ".*HTTP/1%.1"), "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
+	assert((request[1] and find(request[1], ".*HTTP/1%.1")) ~= nil, "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
```
Comment (danielga) [inline lua/websocket/utilities.lua]: I think it makes more sense to have it as `request[1] ~= nil and find(request[1], ".*HTTP/1%.1") ~= nil`.

**Review by danielga — CHANGES_REQUESTED** [review]

File: lua/websocket/utilities.lua (hunk 3 above)
Comment (SwadicalRag) [reply lua/websocket/utilities.lua]: aye aye captain

**Review by danielga — APPROVED** [review]

**Pull request merged**